invenio_github/models.py

```diff
@@ -238,6 +238,9 @@ def enabled(self):
 
     def latest_release(self, status=None):
         """Chronologically latest published release of the repository."""
+        # Bail out fast if object not in DB session.
+        if self not in db.session:
```

Comment:
The object can be binded to a different session and then you can not get latest release.